README.md

```diff
@@ -11,7 +11,7 @@ The Message Format Working Group (MFWG) is tasked with developing an industry st
 
 ## MessageFormat 2 Technical Preview
 
-The MessageFormat 2 specification is a new part of
+The [MessageFormat 2 specification](./spec/) is a new part of
```

Comment:
Technically `/spec/` is not the thing in tech preview. Instead, it's actually in the TR35 report.